internal/cli/cmd/dashboard/dashboard.go

```diff
@@ -0,0 +1,29 @@
+package dashboard
+
+import (
+	"github.com/bacongobbler/browser"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/akuity/kargo/internal/cli/config"
+)
+
+func NewCommand(cfg config.CLIConfig) *cobra.Command {
+	return &cobra.Command{
+		Use:     "dashboard",
+		Short:   "Open the Kargo Dashboard in your default browser.",
+		Example: "kargo logout",
+		RunE: func(*cobra.Command, []string) error {
+			if cfg.APIAddress == "" {
+				return errors.New(
+					"it seems like you are not logged in - Please use kargo login first",
```

Comment:
There's a similar error message one other place in the code.

I think when we re-organize the CLI (which I alluded was coming), we'll make sure to DRY this up, but for now, I just want to make this message match the other.

```suggestion
					"seems like you are not logged in; please use `kargo login` to authenticate",
```